src/api.js

```diff
@@ -1,3 +1,5 @@
+import axios from 'axios';
+
 const apiServer = "https://nc-news-be-proj.onrender.com/api";
 
 const Api = {
```

Comment:
I would avoid the capitalisation convention here - this is not a class or a component - but I like that you've generalised you're endpoints away

Reply:
One thing I'm not too keen on however is the names of your functions in this file - they suggest that they are responsible for making the request rather than simply returning the associated url